README.md

```diff
@@ -27,7 +27,7 @@ _Konstantin, please update this section_
 1. Clone repository with submodules: `git clone --recursive https://github.com/MIPT-ILab/mipt-mips.git`
 1. Install LibELF and Boost ([instruction](https://github.com/MIPT-ILab/mipt-mips/wiki/Required-libraries))
 1. [Build MIPS binutils](https://github.com/MIPT-ILab/mipt-mips/wiki/MIPS-binutils) if you need to build MIPS ELF binaries.
-1. In POSIX environment (Linux, OSX, Windows-Ubuntu, and Msys) you can build MIPT-MIPS just by `make mipt-mips` command.
+1. In POSIX environment (Linux, OSX, Windows-Ubuntu, and Msys) create build directory and build MIPT-MIPS just by `cmake <path_to_mipt_mips_directory>/simulator && make` command ([CMake](http://github.com/MIPT-ILab/mipt-mips/wiki/CMake) is required).
```

Comment:
Could your please be more verbose? I think you may put sequence of commands here.